pkg/client-sdk/client.go

```diff
@@ -205,6 +206,9 @@ func LoadArkClient(sdkStore types.Store) (ArkClient, error) {
 		},
 	}
 
+	// subscribe to boarding address events
+	explorerSvc.SubscribeToAddressEvent(walletSvc.SubscribeAddressEvent(context.TODO()))
+
```

Comment:
this should be moved within the `if cfgData.WithTransactionFeed` block?
for the cli, that is a use case of the sdk **without tx feed**, it's useless to subscribe for addresses event as it won't ever use the subscription.